test/lib/Elastica/Test/Aggregation/AbstractSimpleAggregationTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Elastica\Test\Aggregation;
+
+use Elastica\Aggregation\AbstractSimpleAggregation;
+use Elastica\Exception\InvalidException;
+
+class AbstractSimpleAggregationTest extends BaseAggregationTest
+{
+    public function setUp()
+    {
+        $this->aggregation = $this->getMockForAbstractClass(
+            AbstractSimpleAggregation::class,
```

Comment:
"::class" introduced in PHP 5.5, in composer.json you can find information that Elastica support 5.4, can you fix it please?